Remove devDependencies from package.json as setting npm to
production mode does not seems to work.

diff --git a/third_party/generic_server/package.json b/third_party/generic_server/package.json
index 3d512df9..3f3223ec 100644
--- a/third_party/generic_server/package.json
+++ b/third_party/generic_server/package.json
@@ -51,13 +51,5 @@
 		"watch": "tsc -b -w",
 		"postinstall": "cd client && npm install && cd ../server && npm install && cd ..",
 		"test": "sh ./scripts/e2e.sh"
-	},
-	"devDependencies": {
-		"@types/mocha": "^8.0.3",
-		"@types/node": "^12.12.0",
-		"@typescript-eslint/parser": "^2.3.0",
-		"eslint": "^6.4.0",
-		"mocha": "^8.1.1",
-		"typescript": "^3.9.4"
 	}
 }
diff --git a/third_party/generic_server/client/package.json b/third_party/generic_server/client/package.json
index fa728124..abad9b46 100644
--- a/third_party/generic_server/client/package.json
+++ b/third_party/generic_server/client/package.json
@@ -14,9 +14,5 @@
 	},
 	"dependencies": {
 		"vscode-languageclient": "^6.1.3"
-	},
-	"devDependencies": {
-		"@types/vscode": "1.43.0",
-		"vscode-test": "^1.3.0"
 	}
 }
